flask_webpackext/project.py

```diff
@@ -77,11 +78,29 @@ def storage_cls(self):
         """Get storage class."""
         return current_webpack.storage_cls
 
+    @property
+    @cached
+    def npmpkg(self):
+        """Get API to the package for the configured package manager."""
+        npm_pkg_cls = current_webpack.npm_pkg_cls
+        # assert (
+        #     callable(npm_pkg_cls),
+        #     "FLASK_WEBPACKEXT_NPM_PKG_CLS is expected to be callable",
+        # )
+
+        return npm_pkg_cls(self.path)
+
+
+# backwards compatibility
+_PathStorageMixin = _PathStoragePackageMixin
```

Comment:
i'm not sure if we even want to have this backwards compat, since it's clearly marked as internal already via its naming scheme

Reply:
i think that too, that private variables didn't have to be preserved for backwards compatibility

Reply:
removed